src/com/javabasic/lesson13/homework/Human.java

```diff
@@ -0,0 +1,42 @@
+package com.javabasic.lesson13.homework;
+
+import java.util.Objects;
+
+public class Human {
+
+    private String name;
+    private Human father;
+    private Human mother;
+
+    public Human(String name) {
+        this.name = name;
+    }
+
+    public Human(String name, Human father, Human mother) {
+        this.name = name;
+        this.father = father;
+        this.mother = mother;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public String toString() {
+        if (this.father!=null || this.mother!=null) {
```

Comment:
better to check separately father and mother, because one of them can be null, another not null